src/doc/rustc/theme/pagetoc.css

```diff
@@ -0,0 +1,104 @@
+/* Inspired by https://github.com/JorelAli/mdBook-pagetoc/ */
```

Comment:
@davidtwco @wesleywiser, legal question here, `pagetoc.css` and `pagetoc.js` are both heavily inspired by https://github.com/JorelAli/mdBook-pagetoc/ which is licensed under the "WTFPL" license (Do What The F*ck You Want To Public License).

Is it okay for us to re-license those files under our dual license MIT/Apache-2?

Reply:
Added the licence and the hash we used.